tests/MongoDB.Driver.Tests/UpdateDefinitionBuilderTests.cs

```diff
@@ -637,11 +650,23 @@ private void Assert<TDocument>(UpdateDefinition<TDocument> update, string expect
             Assert(update, BsonDocument.Parse(expected));
         }
 
+        private void AssertThrow<TDocument, TException>(UpdateDefinition<TDocument> update, string errorMessage) where TException : Exception
+        {
+            var exception = Xunit.Assert.Throws<TException>(() => { Render(update); });
```

Comment:
Why aren't you using FluentAssertions' Record.Exceptions like we do everywhere else?

    var exception = Record.Exception(() => Render(update));

Reply:
The XUnit class has a generic parameter which is a little bit more convenient:) Anyway, I've fixed.
Done